src/muz/spacer/spacer_context.cpp

```diff
@@ -840,6 +840,29 @@ reach_fact *pred_transformer::get_used_origin_rf(model& mdl, unsigned oidx) {
     return nullptr;
 }
 
+// get all reachable facts shifted to oidx
```

Comment:
the comment says that formulas over `oidx` are returned, but the function returns formulas over next-variables. 
Only `rf->tag()` is shifted. This seems more a comment issue.